solidity/contracts/bridge/BridgeGovernanceParameters.sol

```diff
@@ -456,6 +464,49 @@ library BridgeGovernanceParameters {
         return self.newDepositTxMaxFee;
     }
 
+    /// @param _newDepositRevealAheadPeriod New deposit reveal ahead period.
```

Comment:
```suggestion
    /// @notice Begins the deposit reveal ahead period update process.
    /// @param _newDepositRevealAheadPeriod New deposit reveal ahead period.
```

This piece of documentation is sometimes available in `begin*` functions in this file and sometimes it's not, but I think we should add it to stay consistent with the majority of functions.